py-pkg/uptasticsearch/aggs_parsers.py

```diff
@@ -0,0 +1,171 @@
+
+import pandas as pd
+
+
+def _terms_agg_to_df(aggs_json):
```

Comment:
can you please create another issue that talks about how the "supported aggs" table in the README needs to get updated to talk about Python?

Reply:
#86